cognite/neat/core/extractors/graph_sheet_to_graph.py

```diff
@@ -2,15 +2,18 @@
 import warnings
 
 import pandas as pd
-from rdflib import RDF, Graph, Literal
+from rdflib import RDF, XSD, Literal, Namespace
 
 from cognite.neat.core.data_classes.transformation_rules import TransformationRules
 
 
-def sheet2graph(
-    graph_capturing_sheet: dict[str, pd.DataFrame], transformation_rule: TransformationRules, separator: str = ","
-):
-    """Converts a graph capturing sheet to an RDF graph
+def sheet2triples(
```

Comment:
Good name improvement. This is more intuitive to me as well.

Reply:
Great !